pages/getting-started.mdx

```diff
@@ -27,21 +27,17 @@ Get Memgraph up and running on your data by following these installation steps.
 
 ### Download and install Memgraph
 
-Memgraph bundles the database along with various analytical tools into distinct
-Docker images. If you're new to Memgraph or you're in a developing stage, we
-recommend using the `memgraph-platform` image. Besides the database, it also
+If you're new to Memgraph or you're in a developing stage, we
+recommend using the Memgraph Paltform. Besides the database, it also
 includes all the tools you might need to analyze your data, such as command-line
 interface [mgconsole](/getting-started/cli), web interface [Memgraph
 Lab](/data-visualization) and a complete set of algorithms within a
-[MAGE](/advanced-algorithms) library. 
+[MAGE](/advanced-algorithms) library.
 
 With the [Docker](https://docs.docker.com/engine/install/) running in the
 background, run the following command in the console:
- 
-<CodeSnippet code="docker run -p 7687:7687 -p 7444:7444 -p 3000:3000 --name memgraph memgraph/memgraph-platform" page="getting-started"/>
 
-You can stop the instance with `docker stop memgraph` and start it with `docker
-start memgraph` commands.
+<CodeSnippet code="curl https://install.memgraph.com | sh" page="getting-started/first-steps-with-memgraph"/>
```

Comment:
This is okay for Linux/MacOS. We can go with dynamic rendering like I did on web, but I am reconsidering that (want to make sure users see all options regardless of their OS used for browsing). Hence, there should be two commands, and be careful with changing the page parameter (I see you changed it and tracking depends on it).
Try creating two options here, one for Linux/MacOS, one for Windows. I will then align all for tracking later.

Reply:
I've fixed the page name. We still don't how will the Windows code look like, but I'll add something

Reply:
I've added a workaround for Windows to all places where we use CodeSnippet. When we will have a Windows script, we will update the docs one more time.